src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -109,7 +131,12 @@ case class MethodConfiguration(
                    ) extends DomainObject {
   def toShort : MethodConfigurationShort = MethodConfigurationShort(name, rootEntityType, methodRepoMethod, namespace)
   def path( workspaceName: WorkspaceName ) = workspaceName.path+s"/methodConfigs/${namespace}/${name}"
-  def idField = "name"
+
+  // a custom idFilterFn is required because two fields are necessary to identify
+  def idField = None
```

Comment:
I'm not sure what `idField` being an option achieves here. It's checked in the default implementation of `idFilterFn`, but tripping that error would require that you set `idField = None` and fail to override `idFilterFn`. Surely you're more likely to do the opposite if you're in a rush - and that'd work fine?